app/build.gradle

```diff
@@ -5,6 +5,10 @@ plugins {
     id 'dagger.hilt.android.plugin'
 }
 
+plugins {
+    id 'com.onesignal.androidsdk.onesignal-gradle-plugin'
+}
+
```

Comment:
@z3bi For some reason, adding this in the `plugins {}` above wasn't working, but this did the trick